src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-password-form/user-password-form.component.html

```diff
@@ -1,121 +1,3 @@
-                     autocomplete="new-password"
-                     formControlName="newpassword">
-              <span class="input-group-append">
-                <button type="button"
-                        class="btn btn-light"
-                        cdPasswordButton="newpassword">
-                </button>
-              </span>
-            </div>
-            <div class="password-strength-level">
-              <div class="{{ passwordStrengthLevelClass }}"
-                   data-toggle="tooltip"
-                   title="{{ passwordValuation }}">
-              </div>
-            </div>
-            <span class="invalid-feedback"
-                  *ngIf="userForm.showError('newpassword', frm, 'required')"
-                  i18n>This field is required.</span>
-            <span class="invalid-feedback"
-                  *ngIf="userForm.showError('newpassword', frm, 'notmatch')"
-                  i18n>The old and new passwords must be different.</span>
-            <span class="invalid-feedback"
-                  *ngIf="userForm.showError('newpassword', frm, 'passwordPolicy')">
-              {{ passwordValuation }}
-            </span>
-          </div>
-        </div>
-
-        <!-- Confirm new password -->
-        <div class="form-group row">
-          <label class="cd-col-form-label required"
-                 for="confirmnewpassword"
-                 i18n>Confirm new password</label>
-          <div class="cd-col-form-input">
-            <div class="input-group">
-              <input class="form-control"
-                     type="password"
-                     autocomplete="new-password"
-                     placeholder="Confirm new password..."
-                     id="confirmnewpassword"
-                     formControlName="confirmnewpassword">
-              <span class="input-group-append">
-                <button class="btn btn-light"
-                        cdPasswordButton="confirmnewpassword">
-                </button>
-              </span>
-            </div>
-            <span class="invalid-feedback"
-                  *ngIf="userForm.showError('confirmnewpassword', frm, 'required')"
-                  i18n>This field is required.</span>
-            <span class="invalid-feedback"
-                  *ngIf="userForm.showError('confirmnewpassword', frm, 'match')"
-                  i18n>Password confirmation doesn't match the new password.</span>
-          </div>
-        </div>
-      </div>
-
-      <div class="card-footer">
-        <cd-form-button-panel (submitActionEvent)="onSubmit()"
-                              [form]="userForm"
-                              [submitText]="(action | titlecase) + ' ' + (resource | upperFirst)"
-                              wrappingClass="text-right"></cd-form-button-panel>
-      </div>
-    </div>
-  </form>
-</div>
+<formio [form]="passwordForm" [options]="formConfig.options" (submit)="onSubmit($event)"></formio>
```

Comment:
this looks amazing :fireworks:

Reply:
Thanks @nizamial09 , Please try this one as well and let me know if there are any issues that you face.